plugin/googlemeet/lang/spanish.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/* For licensing terms, see /license.txt */
+
+$strings['plugin_title'] = 'Hangouts Meet';
+$strings['plugin_comment'] = 'Google Hangouts Meet es el sistema de videoconferencia de Google, Conformado por a la app de Hangouts Meet y el Hardware de videoconferencia.';
+$strings['tool_title'] = 'Nombre de la herramienta';
+$strings['tool_title_help'] = 'Puede cambiar el nombre de la herramienta Google Meet; Ejemplo: Videoconferencia';
+$strings['google_meet_enabled'] = 'Activar Herramienta Google Meet';
+$strings['google_meet_enabled_help'] = 'Escoga si desea activar el plugin en su plataforma Chamilo, esta acción activara una herramienta extra en las herramienta del curso.';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line exceeds maximum limit of 120 characters; contains 174 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5686281)